README.md

```diff
@@ -95,6 +95,15 @@ try:
 except ImportError:
   pass
 ```
+### Slimmen Package
+To remove the tests, information and caches from the installed packages, 
+enable the `slim` option. This will: 1. `strip` the `.so` files, remove `__pycache__` 
```

Comment:
Remove the `1.` if you're not actually gonna have a numbered list :wink:

Reply:
Oh, yeah, thanks a lot! will fix it up :smile: